crates/core/tedge/src/cli/mqtt/subscribe.rs

```diff
@@ -122,25 +111,19 @@ fn subscribe(cmd: &MqttSubscribeCommand) -> Result<(), anyhow::Error> {
                         }
                         n_packets += 1;
                         if matches!(cmd.count, Some(count) if count > 0 && n_packets >= count) {
-                            eprintln!("INFO: Break");
+                            eprintln!("INFO: Received {n_packets} messages");
```

Comment:
Allow for both singular and plural form (without doing the extra logic)

```suggestion
                            eprintln!("INFO: Received {n_packets} message/s");
```

Reply:
done in 30906508844a93cf741398048174360e1f1da43d